controllers/AuthController.js

```diff
@@ -0,0 +1,89 @@
+import models from '../database/models';
+import Helper from '../helpers/Auth';
+import ServerResponse from '../modules';
+
+const { successResponse } = ServerResponse;
+const { BlacklistedToken, User } = models;
+
+/**
+ *
+ *
+ * @export
+ * @class AuthController
+ */
+export default class AuthController {
+  /**
+   *
+   * @static
+   * @param {object} req express request object
+   * @param {object} res express response object
+   * @returns {object} returns user data
```

Comment:
kindly separate your JSDoc block tags

Reply:
I pulled in files from someone else's branch and must have missed this but I will make these changes. Thank you

Reply:
Done